test/test_arcsine.cpp

```diff
@@ -272,7 +271,7 @@ void test_spots(RealType)
     BOOST_CHECK_EQUAL(kurtosis_excess(arcsine_01), -1.5); // 3/2
     BOOST_CHECK_EQUAL(support(arcsine_01).first, 0); //
     BOOST_CHECK_EQUAL(range(arcsine_01).first, 0); //
-    BOOST_MATH_CHECK_THROW(mode(arcsine_01), std::domain_error); //  Two modes at x_min and x_max, so throw instead.
+    BOOST_CHECK_THROW(mode(arcsine_01), std::domain_error); //  Two modes at x_min and x_max, so throw instead.
```

Comment:
Just checking... do we need these changes to the tests, or is it just a side effect of merging in an old branch?

Reply:
I was having compilation issues with `BOOST_MATH_CHECK_THROW` but the version from Boost.Test worked just fine. Since the tests are already using Boost.test for everything else I figured it wasn't a big change.